### PR TITLE
LOLCat-ed Lolcat more, and made it less owoy

### PR DESCRIPTION
**Pull request by EmqsaGit**
the old one had too much "w"s and were not even creative- not that mine are much better, but at least it's not just "w"ed versions of the og ¯\_(ツ)_/¯ PS: forgor line 23, too late

### CODE REVIEW

File: src/main/resources/assets/iris/lang/lol_us.json [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
-  "iris.shaders.toggled": "Toggwed shadews to %s!",
-  "iris.keybind.reload": "Wewoad shadews",
-  "iris.keybind.shaderPackSelection": "Shadewpack sewection scween",
-  "iris.keybind.toggleShaders": "Toggwe Shadews",
-  "iris.keybinds": "Iwis",
-  "iris.shaders.reloaded.failure": "Faiwed to wewoad shadews! Reason: %s",
-  "iris.shaders.toggled.failure": "Faiwed to toggwe shadews! Reason: %s",
-  "iris.sodium.failure.title": "Iwis faiwed to woad!",
-  "iris.sodium.failure.reason": "You have Sodium in youw mods fowdew, howevew Iwis incwudes it. Pwease cwose da game, and wemove Sodium from youw mods fowdew.",
+  "iris.shaders.reloaded": "Shaedews wewoded!",
```
Comment (FoundationGames) [inline src/main/resources/assets/iris/lang/lol_us.json:2]: "Yr shadurz r relodid!"

File: src/main/resources/assets/iris/lang/lol_us.json [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
+  "iris.shaders.toggled": "Turnd Blek cat un grownd to %s!",
```
Comment (FoundationGames) [inline src/main/resources/assets/iris/lang/lol_us.json:3]: "I can has shadur %s!" for toggle and
"Loded ur shadurz again!"

File: src/main/resources/assets/iris/lang/lol_us.json [hunk 3]
```diff
@@ -1,33 +1,33 @@
...
+  "iris.keybind.reload": "Rewoaded Blek cat un gownd",
+  "iris.keybind.shaderPackSelection": "Blek Ket On Grownd Bramd Selectur scren",
```
Comment (FoundationGames) [inline src/main/resources/assets/iris/lang/lol_us.json:5]: "Screen fur Selektin ur Shadur Packz"

File: src/main/resources/assets/iris/lang/lol_us.json [hunk 4]
```diff
@@ -1,33 +1,33 @@
...
+  "iris.keybind.toggleShaders": "Toggwe Blek Cat Un Ground",
```
Comment (FoundationGames) [inline src/main/resources/assets/iris/lang/lol_us.json:6]: "Togle Shadurz"

File: src/main/resources/assets/iris/lang/lol_us.json [hunk 5]
```diff
@@ -1,33 +1,33 @@
...
+  "iris.keybinds": "Iris",
+  "iris.shaders.reloaded.failure": "COWLDNT REWOAD BLECK KET ON GROWND! Reson: %s",
```
Comment (FoundationGames) [inline src/main/resources/assets/iris/lang/lol_us.json:8]: "I cant has relodid shadurz! Reazn: %s"
"I cant has togeld shadurz! Reazn: %s"

File: src/main/resources/assets/iris/lang/lol_us.json [hunk 6]
```diff
@@ -1,33 +1,33 @@
...
+  "iris.shaders.toggled.failure": "COWLDNT FIND BLECK KET TO PUT UN GROWND! Reson: %s",
+  "iris.sodium.failure.title": "Iris cowld nut be able to woad!",
```
Comment (FoundationGames) [inline src/main/resources/assets/iris/lang/lol_us.json:10]: "Iris loding dint wurk!"

File: src/main/resources/assets/iris/lang/lol_us.json [hunk 7]
```diff
@@ -1,33 +1,33 @@
...
+  "iris.sodium.failure.reason": "You hab Sodium in kiten's mods fowder, however Iris includes it. Pleas cwose da game, and rembve Sodium from kitten's mods fowder.",
```
Comment (FoundationGames) [inline src/main/resources/assets/iris/lang/lol_us.json:11]: This translation is outdated, replace with the `notFound` and `incompatible` translations
"Iris needz Sodium fr gud purrformans, but Sodium iznt heere. Plz downlod Sodium, put it in yr mods foldur, n relawnch da game."
"Iris needz Sodium fr gud purrformans, but yr vershun iznt compatibul. Plz downlod Sodium, put it in yr mods foldur, n relawnch da game."

File: src/main/resources/assets/iris/lang/lol_us.json [hunk 8]
```diff
@@ -1,33 +1,33 @@
...
-  "options.iris.refresh": "Refwesh",
-  "options.iris.openShaderPackFolder": "Open Shadew Pack Fowdew...",
-  "options.iris.refreshShaderPacks": "Refwesh Shadew Packs",
-  "options.iris.shaderPackSelection": "Shadew Packs...",
-  "options.iris.shaderPackSelection.title": "Shadew Packs",
-  "options.iris.shaderPackSelection.addedPack": "Added Shadew Pack \"%s\"",
-  "options.iris.shaderPackSelection.addedPacks": "Added %s Shadew Packs",
-  "options.iris.shaderPackSelection.failedAdd": "Fiwes wewe not vawid Shadew Packs",
-  "options.iris.shaderPackSelection.failedAddSingle": "\"%s\" ish not a vawid Shadew Pack",
+  "options.iris.apply": "DUN YET?!",
```
Comment (FoundationGames) [inline src/main/resources/assets/iris/lang/lol_us.json:13]: "try it owt"

File: src/main/resources/assets/iris/lang/lol_us.json [hunk 9]
```diff
@@ -1,33 +1,33 @@
...
+  "options.iris.refresh": "Reflesh",
```
Comment (FoundationGames) [inline src/main/resources/assets/iris/lang/lol_us.json:14]: "mak it fresh"

File: src/main/resources/assets/iris/lang/lol_us.json [hunk 10]
```diff
@@ -1,33 +1,33 @@
...
+  "options.iris.openShaderPackFolder": "Open Blek Ket Veriants Fowder...",
```
Comment (FoundationGames) [inline src/main/resources/assets/iris/lang/lol_us.json:15]: "see yr shadurz foldur"
"mak yr shadur pakz fresh"
"Shadur Packz..." (consistency with resource pack button)
"Shadur Packz"
"I can has shadur pack \"%s\""
"I can has %s shadur packz"
"Dees rnt reel shadur packz!!"
"\"%s\" iznt a reel shadur pack!!"
"Cant hav dis shadur pack"

Hopefully keep the theme for other names

File: src/main/resources/assets/iris/lang/lol_us.json [hunk 11]
```diff
@@ -1,33 +1,33 @@
...
+  "options.iris.shaderPackSelection.addedPacks": "Aded %s Blek Kets",
+  "options.iris.shaderPackSelection.failedAdd": "Deez kitens were not real blek kets",
+  "options.iris.shaderPackSelection.failedAddSingle": "\"%s\" is not real blek ket",
   "options.iris.shaderPackSelection.copyError": "Couwd not add Shadew Pack \"%s\"",
-  "options.iris.shaderPackSelection.copyErrorAlreadyExists": "\"%s\" ish awwready in youw Shadew Packs fowdew!",
-  "options.iris.shaders.disabled": "Shadews: Disabwed",
-  "options.iris.shaders.enabled": "Shadews: Enabwed",
-  "options.iris.shadowDistance": "Mwax Swadow Dwistance",
+  "options.iris.shaderPackSelection.copyErrorAlreadyExists": "\"%s\" is alredy ur blek ket!",
+  "options.iris.shaders.disabled": "Blek Kets: Bai",
```
Comment (FoundationGames) [inline src/main/resources/assets/iris/lang/lol_us.json:25]: "Shadurz: yiss"
"Shadurz: naw"

File: src/main/resources/assets/iris/lang/lol_us.json [hunk 12]
```diff
@@ -1,33 +1,33 @@
...
+  "options.iris.shaders.enabled": "Blek Kets: Hewo",
+  "options.iris.shadowDistance": "Mex blek ketz distamce",
```
Comment (FoundationGames) [inline src/main/resources/assets/iris/lang/lol_us.json:27]: "how far yr shadow goez"

File: src/main/resources/assets/iris/lang/lol_us.json [hunk 13]
```diff
@@ -1,33 +1,33 @@
...
+  "options.iris.shaderPackSelection.addedPack": "Aded Blek Ket \"%s\"",
+  "options.iris.shaderPackSelection.addedPacks": "Aded %s Blek Kets",
+  "options.iris.shaderPackSelection.failedAdd": "Deez kitens were not real blek kets",
+  "options.iris.shaderPackSelection.failedAddSingle": "\"%s\" is not real blek ket",
   "options.iris.shaderPackSelection.copyError": "Couwd not add Shadew Pack \"%s\"",
-  "options.iris.shaderPackSelection.copyErrorAlreadyExists": "\"%s\" ish awwready in youw Shadew Packs fowdew!",
-  "options.iris.shaders.disabled": "Shadews: Disabwed",
-  "options.iris.shaders.enabled": "Shadews: Enabwed",
-  "options.iris.shadowDistance": "Mwax Swadow Dwistance",
+  "options.iris.shaderPackSelection.copyErrorAlreadyExists": "\"%s\" is alredy ur blek ket!",
```
Comment (FoundationGames) [inline src/main/resources/assets/iris/lang/lol_us.json:24]: "U alredy got \"%s\" in yr shadur packz foldur!!"

File: src/main/resources/assets/iris/lang/lol_us.json [hunk 14]
```diff
@@ -1,33 +1,33 @@
...
+  "options.iris.shaders.disabled": "Blek Kets: Bai",
+  "options.iris.shaders.enabled": "Blek Kets: Hewo",
+  "options.iris.shadowDistance": "Mex blek ketz distamce",
   "options.iris.shadowDistance.enabled": "Awwows you to change da mwaximum dwistance fow swadows. Tewwain and entities beyond dis dwistance wiww not cast swadows. Lowering da shadow distance can sigwificantlwy incwease pewfowmance.",
   "options.iris.shadowDistance.disabled": "Youw cuwwent shadew pack has awweady set a wendew dwistance fow swadows; you cawwot chwange wit.",
 
-  "pack.iris.select.title": "Sewect",
-  "pack.iris.list.label": "+ Dwag and Dwop Shadew Packs to add"
+  "pack.iris.select.title": "ME!?", \\wait, what is this for?
```
Comment (FoundationGames) [inline src/main/resources/assets/iris/lang/lol_us.json:31]: select.title -> "Selekt"
configure.title -> "Cunfigur"

File: src/main/resources/assets/iris/lang/lol_us.json [hunk 15]
```diff
@@ -1,33 +1,33 @@
...
+  "pack.iris.list.label": "+ Adopt Blek Kets And put em in dis box"
```
Comment (FoundationGames) [inline src/main/resources/assets/iris/lang/lol_us.json:32]: "+ Drag n drawp yr shadur packz n heer"

**Review by FoundationGames — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/iris/lang/lol_us.json (hunk 15 above)
Comment (EmqsaGit) [reply src/main/resources/assets/iris/lang/lol_us.json:32]: sure

File: src/main/resources/assets/iris/lang/lol_us.json (hunk 14 above)
Comment (EmqsaGit) [reply src/main/resources/assets/iris/lang/lol_us.json:31]: eh

File: src/main/resources/assets/iris/lang/lol_us.json (hunk 13 above)
Comment (EmqsaGit) [reply src/main/resources/assets/iris/lang/lol_us.json:24]: nice

File: src/main/resources/assets/iris/lang/lol_us.json (hunk 12 above)
Comment (EmqsaGit) [reply src/main/resources/assets/iris/lang/lol_us.json:27]: nice

File: src/main/resources/assets/iris/lang/lol_us.json (hunk 11 above)
Comment (EmqsaGit) [reply src/main/resources/assets/iris/lang/lol_us.json:25]: consistent!

File: src/main/resources/assets/iris/lang/lol_us.json (hunk 10 above)
Comment (EmqsaGit) [reply src/main/resources/assets/iris/lang/lol_us.json:15]: very

File: src/main/resources/assets/iris/lang/lol_us.json (hunk 8 above)
Comment (EmqsaGit) [reply src/main/resources/assets/iris/lang/lol_us.json:13]: ur so much better

File: src/main/resources/assets/iris/lang/lol_us.json (hunk 7 above)
Comment (EmqsaGit) [reply src/main/resources/assets/iris/lang/lol_us.json:11]: lol

**Comment by coderbot16** [comment]
This PR appears to be abandoned? Let me know if it should be reopened.

**Pull request closed without merging**